modules/cluster/main.tf

```diff
@@ -54,6 +54,33 @@ module "vpc" {
   }
 }
 
+resource "null_resource" "destroy_lb_sg" {
```

Comment:
We should not destroy resources created outside terraform, the thing that comes to mind is the network load balancer created by helm charts, that needs to be removed before you do terraform destroy. However, if you create the nginx charts using this module instead of versionstream charts, that should not be an issue. Do you still encounter an issue?
I do recall seeing some left over EBS volumes, but I believe those are created by tekton, and should be fixed in tekton (or using EBS lifecycle management, which is beyond the scope of this module), wdyt?

Reply:
> One is created with terraform nginx

That should get destroyed cleanly.

> https://github.com/jx3-gitops-repositories/jx3-eks-vault.

I should update the docs, u need to use only one, and not both, so if you use nginx from terraform, dont use the ones defined here: https://github.com/jx3-gitops-repositories/jx3-eks-vault/tree/main/versionStream/charts/ingress-nginx/ingress-nginx

>When you try to destroy infrastructure you can't do that , because VPC has dependencies on LB and SecurityGroup which were created outside the Terraform

Yes! That is why the idea was to create the loadbalancer in terraform, and not using jx3-eks-vault, again bad documentation on my part ... will add it to docs soon ...